docs/topics/mpp/multiplatform.md

```diff
@@ -7,68 +7,70 @@
 Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the
  same code for [different platforms](mpp-supported-platforms.md) while retaining the flexibility and benefits of native programming. 
 
-This is how Kotlin Multiplatform works.
+## How Kotlin Multiplatform works
 
 ![Kotlin Multiplatform](kotlin-multiplatform.png){width=500}
 
-*   **Common Kotlin** includes the language, core libraries, and basic tools. Code written in common Kotlin works 
+* **Common Kotlin** includes the language, core libraries, and basic tools. Code written in common Kotlin works 
 everywhere on all platforms.
-*   With Kotlin Multiplatform libraries, you can reuse the multiplatform logic in common and platform-specific code. 
+* With Kotlin Multiplatform libraries, you can reuse the multiplatform logic in common and platform-specific code. 
 Common code can rely on a set of libraries that cover everyday tasks such as [HTTP](https://ktor.io/clients/http-client/multiplatform.html), [serialization](https://github.com/Kotlin/kotlinx.serialization), and [managing 
 coroutines](https://github.com/Kotlin/kotlinx.coroutines).
-*   To interop with platforms, use platform-specific versions of Kotlin. **Platform-specific versions of Kotlin** 
+* To interop with platforms, use platform-specific versions of Kotlin. **Platform-specific versions of Kotlin** 
 (Kotlin/JVM, Kotlin/JS, Kotlin/Native) include extensions to the Kotlin language, and platform-specific libraries and tools. 
-*   Through these platforms you can access the **platform native code** (JVM, JS, and Native) and leverage all native
+* Through these platforms you can access the **platform native code** (JVM, JS, and Native) and leverage all native
  capabilities.
 
+## Code sharing between platforms
```

Comment:
```suggestion
## Code sharing among platforms
```

Reply:
Could you motivate this suggestion?

Reply:
When analyzing the HMPP content, I noticed that we commonly use 'share code among platforms' (when there could a lot of them) but 'share code between iOS and Android' (when thers're just 2), following that old rule about the number of objects
I guess we can leave it to editors or keep 'between' if it necessary for SEO